EIPS/eip-4445.md

```diff
@@ -0,0 +1,179 @@
+
+Having a `consumer` role will enable protocols to integrate and build on top of dApps that issue ERC-721 tokens.
+
+Example of kinds of contracts and applications that can benefit from this standard are predominantly metaverses that
+have land and other types of digital assets in those metaverses (scene deployment on land, renting
+land/characters/clothes/passes to events etc.)
+
+## Specification
+
+The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and
+“OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+Every contract compliant to the `ERC721Consumable` extension MUST implement the `IERC721Consumable` interface. The **consumer extension** is OPTIONAL for ERC-721 contracts.
+
+```solidity
+/// @title ERC-721 Consumer Role extension
+///  Note: the ERC-165 identifier for this interface is 0x953c8dfa
+interface IERC721Consumable /* is ERC721 */ {
+
+    /// @notice Emitted when `owner` changes the `consumer` of an NFT
+    /// The zero address for consumer indicates that there is no consumer address
+    /// When a Transfer event emits, this also indicates that the consumer address
+    /// for that NFT (if any) is set to none
+    event ConsumerChanged(address indexed owner, address indexed consumer, uint256 indexed tokenId);
+
+    /// @notice Get the consumer address of an NFT
+    /// @dev The zero address indicates that there is no consumer
+    /// Throws if `_tokenId` is not a valid NFT
+    /// @param _tokenId The NFT to get the consumer address for
+    /// @return The consumer address for this NFT, or the zero address if there is none
+    function consumerOf(uint256 _tokenId) view external returns (address);
+
+    /// @notice Change or reaffirm the consumer address for an NFT
+    /// @dev The zero address indicates there is no consumer address
+    /// Throws unless `msg.sender` is the current NFT owner, an authorised
+    /// operator of the current owner or approved address
+    /// Throws if `_tokenId` is not valid NFT
+    /// @param _consumer The new consumer of the NFT
+    function changeConsumer(address _consumer, uint256 _tokenId) external;
+}
+```
+
+Every contract implementing the `ERC721Consumable` extension is free to define the permissions of a `consumer` (e.g. what
+are consumers allowed to do within their system) with only one exception - consumers MUST NOT be considered owners,
+authorised operators or approved addresses as per the ERC-721 specification. Thus, they MUST NOT be able to execute
+transfers & approvals.
+
+The `consumerOf(uint256 _tokenId)` function MAY be implemented as `pure` or `view`.
+
+The `changeConsumer(address _consumer, uint256 _tokenId)` function MAY be implemented as `public` or `external`.
+
+The `ConsumerChanged` event MUST be emitted when a consumer is changed.
+
+On every `transfer`, the consumer MUST be changed to the zero address the same way `approve` address is changed
+
```

Comment:
Hi! This is a pretty well written ERC from what I see. I'd recommend including EIP165 details here - that contracts implementing this ERC must respond `true` when `supportsInterface` is called with `0x953c8dfa`.